prime-router/settings/prod/0029-direct.yml

```diff
@@ -66,7 +66,7 @@
     - "matches(sender_id, SafeHealth,ImageMover,Cue,InBios)"
     qualityFilter:
     - "allowAll()"
-    - "doesNotMatch(processing_mode_code,T,D)"
+    - "doesNotMatch(processing_mode_code,D)"
     reverseTheQualityFilter: false
```

Comment:
This change is in support of some testing we want to do, comparing hhsprotect sends from waters vs from ReportStream.  In order to do that I need to allow HHSProtect to take Training data.